aries-site/src/pages/components/layer/center-layer.mdx

```diff
@@ -0,0 +1,45 @@
+import { Notification } from 'grommet';
+import { Example } from '../../../layouts';
+import { CenterLayerAnatomy, ContentDrivenLayout } from '../../../examples';
+
+## Anatomy
+
+This diagram highlights details specific to center layers. See [general layer anatomy](/components/layer#anatomy) for specs that apply to all layers.
+
+<Example plain>
+  <CenterLayerAnatomy />
+</Example>
+
+| Label | Region             | Required?                                  | Purpose                                                           | Notes                                                                                                                               |
+| ----- | ------------------ | ------------------------------------------ | ----------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| **1** | **Close button**   | Required on layers with no footer actions. | Closes the layer.                                                 | Use "FormClose" icon.                                                                                                               |
+| **2** | **Footer actions** | Required on actionable layers.             | Provides actions for user to accept layer consequences or cancel. | The actions should be right-aligned with primary action on the right. One action should be a "Cancel" button that closes the layer. |
```

Comment:
Should there be a link to the actionable layers section?

Reply:
Grommet markdown table has a bug where it places a new line after an anchor. Agree to linking but proposing we resolve the grommet bug first.